data/editor.config

```diff
@@ -363,86 +358,81 @@
         <include field="operator" />
       </type>
       <type id="amenity-car_wash">
-        <include group="poi" />
-        <include field="internet" />
+        <include group="poi_internet" />
       </type>
       <type id="amenity-childcare">
-        <include group="poi" />
-        <include field="internet" />
+        <include group="poi_internet" />
       </type>
       <type id="amenity-veterinary" group="health">
-        <include group="poi" />
-        <include field="internet" />
+        <include group="poi_internet" />
       </type>
       <type id="amenity-charging_station">
         <include group="poi" />
         <include field="operator" />
       </type>
       <type id="craft" can_add="no">
-        <include group="poi" />
+        <include group="poi_internet" />
       </type>
       <type id="craft-beekeeper">
-        <include group="poi" />
+        <include group="poi_internet" />
       </type>
       <type id="craft-blacksmith">
-        <include group="poi" />
+        <include group="poi_internet" />
       </type>
       <type id="craft-brewery" group="shop">
-        <include group="poi" />
+        <include group="poi_internet" />
       </type>
       <type id="craft-carpenter">
-        <include group="poi" />
+        <include group="poi_internet" />
       </type>
       <!-- Not addable because ambigous with shop=confectionery -->
       <type id="craft-confectionery" group="shop" can_add="no">
-        <include group="poi" />
+        <include group="poi_internet" />
       </type>
       <type id="craft-electrician">
-        <include group="poi" />
+        <include group="poi_internet" />
       </type>
       <type id="craft-electronics_repair">
-        <include group="poi" />
+        <include group="poi_internet" />
       </type>
       <type id="craft-gardener" can_add="no">
-        <include group="poi" />
+        <include group="poi_internet" />
       </type>
       <type id="craft-handicraft">
-        <include group="poi" />
+        <include group="poi_internet" />
       </type>
       <type id="craft-hvac" group="shop">
-        <include group="poi" />
+        <include group="poi_internet" />
       </type>
       <type id="craft-metal_construction" can_add="no">
-        <include group="poi" />
+        <include group="poi_internet" />
       </type>
       <type id="craft-painter" can_add="no">
```

Comment:
Why some are marked can_add="no"?

Reply:
Maybe this https://github.com/organicmaps/organicmaps/pull/2747#issuecomment-1156538277